src/schema/contentful/phoenix.js

```diff
@@ -76,6 +76,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
+    "The type of employer this person is"
+    type: String
+    "Current working status"
```

Comment:
On the same note as above, maybe we can replace this with
```suggestion
    "Is their relationship with the organization still active?"
```